#39 map switching

MMCQ33 opened the pull request.

MMCQ33 committed:
Added minimap clicks

Made minimap clickaable and now it also sends you to that location. Cleaned up a little bit of code.

MMCQ33 committed:
Merge branch 'Develop' into #39-MapSwitching

MMCQ33 committed:
fixed movement

fixed not moving when trying to go outside of the bounding box on one axis but moving the other along the other axis aswell.

ErikLouwerse reviewed (changes requested):
change vars to actual types pls

MMCQ33 committed:
fixed comments and added comments in code

fixed vars towards types, added comments in the code to demostrate what it does.

jetenergy commented on Hivemind/Assets/Scripts/Ui/UiController.cs:
```diff
@@ -1,19 +1,24 @@
 
     [SerializeField] private GameObject mindBuilderPanel;
 
+    [SerializeField] private GameObject mainCamera;
```
Deze kan je vinden aangezien er maar 1 maincamera object per scene aanwezig is, heb je ook minder problemen met een nieuwe map aanmaken waar je dit handmatig moet invullen

jetenergy replied:
dit doe je namelijk ook met minimaps

MMCQ33 committed:
fixed order of awake and start, main camera gets grabbed

main camera is not a serilized field and order and awake and start is now at the beginning
